src/colors/color.rs

```diff
@@ -337,6 +337,48 @@ impl Color {
         (long, medium, short)
     }
 
+    /// convert normalized RGB value to linear value
+    fn to_linear(u: f32) -> f32 {
```

Comment:
```suggestion
    fn to_linear(value: f32) -> f32 {
```
Please refrain from using single character variables (except for color variables). They can make the code hard to read and do not provide any meaning to the reader.